packages/common/src/CondaEnvWidget.tsx

```diff
@@ -54,9 +54,9 @@ export class CondaEnvWidget extends ReactWidget {
   /**
    * Conda environment Manager
    */
-  private _envModel: IEnvironmentManager;
+  protected _envModel: IEnvironmentManager;
   /**
    * Signal triggering a React rendering if widget is resized
    */
-  private _resizeSignal = new Signal<CondaEnvWidget, ISize>(this);
+  protected _resizeSignal = new Signal<CondaEnvWidget, ISize>(this);
```

Comment:
another option style-wise: adding protected getter functions to _envModel & _resizeSignal  
 ```
 /**
   * Conda environment Manager
   */
  private _envModel: IEnvironmentManager;
  /**
   * Signal triggering a React rendering if widget is resized
   */
  private _resizeSignal = new Signal<CondaEnvWidget, ISize>(this);

  public get envModel() : IEnvironmentManager {
    return this._envModel;
  }
  
  public get resizeSignal() : Signal<CondaEnvWidget, ISize> {
    return this._resizeSignal;
  }
```
lets us keep _envModel & _resizeSignal `private`

Reply:
Ahh, this is probably a better approach. Thanks!

Reply:
So after trying to do this with getters and setters, I ran into an issue. In the child `CondaStoreEnvWidget._resizeSignal` needs to be of type `Signal<CondaStoreEnvWidget, ISize>`. I can't assign that type to that attribute if it's private. So I'll keep this as is for now, until I learn a better way.